frontend/device/android/device.lua

```diff
@@ -60,6 +60,11 @@ function Device:init()
         self.isTouchDevice = yes
     end
 
+    -- check if we disabled support for wakelocks
+    if G_reader_settings:isTrue("disable_wakelock") then
```

Comment:
Since the setting is Android-specific, should we make that clear in the setting name? Pinging @poire-z

Reply:
I wouldn't mind, as only Android would get that setting. But `disable_android_wakelock` sounds explicite. Or `use_android_wakelocks=false`

Reply:
Yeah, let's go for `disable_android_wakelock`. :+1: